src/BitmapText.cpp

```diff
@@ -770,12 +772,12 @@ void BitmapText::DrawPrimitives()
 		std::vector<RageVector3> vGlyphJitter;
 		if( m_bJitter )
 		{
-			int iSeed = std::lrint( RageTimer::GetTimeSinceStartFast()*8 );
-			RandomGen rnd( iSeed );
+			float rnd1 = GetRandomFloat();
+			float rnd2 = GetRandomFloat();
 
 			for( unsigned i=0; i<m_aVertices.size(); i+=4 )
 			{
-				RageVector3 jitter( rnd()%2, rnd()%3, 0 );
+				RageVector3 jitter( rnd1, rnd2, 0 );
```

Comment:
I think can just in place it here.

Reply:
I wasn't sure to keep the modulo or not so I did for now